src/commands/publish/publishHandler.ts

```diff
@@ -37,15 +39,18 @@ export const publishHandler: (...args: any[]) => void | Promise<void> = async (
   const network = options.network ? findNetworkByName(options.network) : await networkSelectionPrompt();
   const maintainer = options.maintainer ?? (await addressPrompt(strings.MAINTAINER_ADDRESS));
 
-  console.table({
-    setupContract,
-    subdomain: `${subdomain}.plugin.dao.eth`,
-    chain: network.name,
-    maintainer,
-  });
+  logTable([
+    { setupContract },
+    { subdomain: `${subdomain}.plugin.dao.eth` },
```

Comment:
This is tricky because - as far as I'm concerned - subdomains only work on mainnet. Let's double check here with OS though and fix if needed

Reply:
hmmm what do you mean by they only work on mainnet? The canonical ENS deployment is on mainnet but we deploy our own version on the other networks we are on

Reply:
I don't think we do actually.. let's confirm this with OS.

Also, even if we do, there's no way for devs to use this address and have it resolve to their plugin, so it's a tricky UX.

I suggest we add some copy regarding this, potentially at the bottom after the table shows up - sharing something like: "Keep in mind that your plugin's subdomain will only be resolved in mainnet, as is the only network currently supported by ENS"

Reply:
ive added it but if happy to just remove the field if it makes it clearer

Reply:
I'd add a conditional statement here -  this should only come up if the network supports subdomains. If the network doesn't, then the value for `subdomain` should be: "%{network.name} does not yet support ENS subdomains."